maths/series/p_series.py

```diff
@@ -10,29 +10,45 @@
 """
 
 
-def p_series(nth_term: int, power: int) -> list[float]:
+from typing import Union
+
+
+def p_series(
+    nth_term: Union[int, float, str], power: Union[int, float, str]
+) -> Union[list[str], str]:
```

Comment:
Let's not confuse the caller.  Let's pick one and only deliver them a single data type.  I think a list of strings even if that list only has one string is the least surprising.

Reply:
So put that string in [ ] ?